css/main.css

```diff
@@ -0,0 +1,74 @@
+h1,h1+p{
+    color:#333333;
+
+}
+h1 {
+    font-size:40px;
+    font-weight:bold;
+}
+h1+p{
+    font-size:20px;
+}
+body {
+    background-color:#EEEEEE;
+    font-family:'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+    color:#333333;
+
+}
+h1,h1+p,.landpage,.comment,.final,.footer{
+    text-align: center;
+
+}
+.landpage{
+    background-color:white;
+    margin:0px 20px;
+}
+img{
+    max-width:100%;
+    display:block;
+    margin: 0px auto;
+}
+.landpage figcaption{
+    font-size:20px;
+    padding-bottom:20px;
+}
+.section h2{
+    padding: 40px;
+}
+.section ul span{
+    font-weight: bold;
+
+}
+.section ul {
+    font-size:27px;
+    width: 50%;
+    margin-left:25%;
+    
+}
+.section ul li{
+    padding:15px;
+}
+.section h2{
+    text-align: center;
+    
+}
+.section .comment{
+    font-style: italic;
+    font-size:27px;
+    width: 50%;
+    margin-left:25%;
+    text-align: left;
+    color:#333;
+    margin-top:50px;
+    margin-bottom:50px;
+
+
+}
+.final{
+    font-size:25px;
+    font-weight: bold;
+}
+.footer{
+    background-color:white;
+    height:3vh;
+}
```

Comment:
you have a lot of linter errors please check your CSS code here are some of the errors: -
h1,h1+p{
    color:#333333;

}
should be : - 
h1 , h1 + p {
  color: #333333;
}
as you see you should add space between selector and { or when selecting more than one selector you should add space before and after,
you should put space after: also the property should  2 spaces indentation inside the css rule 
between every rule there is a new line in between like 
.section ul li {
  padding: 15px;
}

.section h2 {
  text-align:  center;
}

all of these are important for clearness and readability